Makefile

```diff
@@ -163,21 +175,22 @@ install : $(libceed) $(OBJDIR)/ceed.pc
 	$(INSTALL_DATA) $(libceed) "$(DESTDIR)$(libdir)/"
 	$(INSTALL_DATA) $(OBJDIR)/ceed.pc "$(DESTDIR)$(pkgconfigdir)/"
 
-.PHONY: all cln clean print test tst examples astyle install doc
+.PHONY : all cln clean print test tst prove prv examples astyle install doc
+
 cln clean :
 	$(RM) *.o *.d $(libceed)
 	$(RM) -r *.dSYM $(OBJDIR) $(LIBDIR)/pkgconfig
 	$(MAKE) -C examples clean
 	$(MAKE) -C examples/mfem clean
-	cd examples/nek5000; bash make-nek-examples.sh clean; cd ../..;
+	cd examples/nek5000; make-nek-examples.sh clean; cd ../..
```

Comment:
Why would that script be in PATH?

There is already a makefile in that directory. Why doesn't `make clean` make clean?